src/gui/sign.sh

```diff
@@ -1,14 +1,23 @@
 gui_sign() {
     local file output
-    file=$(yad --file --title="EasyGnuPG | Sign a File")
 
+    file=$(yad --file --title="EasyGnuPG | Sign a File")
+    [[ -n "$file" ]] || return 0
     if [[ -f "$file.signature" ]]; then
-        yesno "$file.signature already exists. Do you want to overwrite??" || exit 0
+        yesno "File already exists:\n<tt>$file.signature</tt>\n\nDo you want to overwrite it?" || return 0
+        rm -f "$file.signature"
     fi
-    
+
     output=$(call cmd_sign $file)
     [[ -n "$output" ]] && message error "$output"
-    [[ -f "$file.signature" ]] && message info "Signature saved as $file.signature"
+
+    if [[ -s "$file.signature" ]]; then
+        yad --file --filename="$file.signature" &
```

Comment:
Hi, I didn't understand the purpose of line 15.
Is it just showing the file to the user? Or you wanted something like user being able to select the desired signature file?